client/src/components/graph/drawPointsRegl.js

```diff
@@ -38,7 +39,20 @@ export default function(regl) {
     uniforms: {
       distance: regl.prop("distance"),
       view: regl.prop("view"),
-      projection: ({viewportWidth, viewportHeight}) => mat4.perspective([], Math.PI / 2, viewportWidth / viewportHeight, 0.01, 1000)
+      projection: ({ viewportWidth, viewportHeight }) => {
+        const aspectRatio = viewportWidth / viewportHeight;
+        let m = mat4.perspective(
+          [],
+          Math.PI / 2,
+          viewportWidth / viewportHeight,
+          0.01,
+          1000
+        );
+        if (aspectRatio < 1) {
+          m = mat4.scale(m, m, mat3.fromValues(1, 1, 1 / aspectRatio));
```

Comment:
similarly here should be `m = mat4.scale(m, m, vec3.fromValues(1, 1, 1 / aspectRatio));` if you make the change above

Reply:
will fix